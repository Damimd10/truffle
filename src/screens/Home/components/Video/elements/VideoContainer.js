import styled from 'styled-components';

const VideoContainer = styled.section`
  width: 700px;
`;

VideoContainer.displayName = 'VideoContainer';

export default VideoContainer;
