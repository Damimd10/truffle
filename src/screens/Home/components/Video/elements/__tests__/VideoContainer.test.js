import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import VideoContainer from '../VideoContainer';

describe('Button Style Element', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<VideoContainer />);
  });

  it('should create a style structure for the component', () =>
    expect(toJson(wrapper)).toMatchSnapshot());
});
