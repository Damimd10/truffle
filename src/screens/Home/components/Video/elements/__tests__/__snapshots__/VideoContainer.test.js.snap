// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button Style Element should create a style structure for the component 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": true,
        "rules": Array [
          "
  width: 700px;
",
        ],
      },
      "displayName": "VideoContainer",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "section",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;
